core/modules/replica/DatabaseServices.cc

```diff
@@ -167,6 +167,25 @@ json TransactionInfo::toJson() const {
 }
 
 
+json TransactionContribInfo::toJson() const {
+    json info;
+    info["id"] = id;
+    info["transaction_id"] = transactionId;
+    info["worker"]     = worker;
+    info["database"]   = database;
+    info["table"]      = table;
+    info["chunk"]      = chunk;
+    info["overlap"]    = isOverlap ? 1 : 0;
```

Comment:
just curious, where does `isOverlap` come from? I don't see it anywhere else in this file.

Reply:
This is a data member of the class (see the class's header `core/modules/replica/DatabaseServices.h`):
```
class TransactionContribInfo {
public:
    ...
    bool isOverlap = false;     /// (optional) A flavor of the chunked table (partitioned tables only)
```

Reply:
lol, thx. I've got 🐍-brain I guess.